Update Zen References & Thread Names

hellcatz opened the pull request:
Update ZenCash references to resolve #92, resolve #72

dkaidalov commented:
I just want to warn that this change is basically a hard fork and could lead to the network split. More work should be done to roll out it safely.

kendricktan commented:
Yep, thats on the development roadmap. https://trello.com/c/jga5LDyP/37-nulltransaction-back-to-protocol

hellcatz committed:
Update common.h

hellcatz committed:
Update httpserver.cpp

hellcatz committed:
Update init.cpp

Update Zen references
Remove extra white spaces resolve https://github.com/ZencashOfficial/zen/issues/72

hellcatz committed:
Update main.cpp

hellcatz committed:
Update metrics.cpp

hellcatz committed:
Update rpcblockchain.cpp

hellcatz committed:
Update rpcwallet.cpp

hellcatz committed:
Update walletdb.cpp

hellcatz committed:
Update rpcwallet.cpp
